resources/lib/ui/embed_extractor.py

```diff
@@ -97,7 +97,7 @@ def playSource():
         reqObj = http.send_request(playUrl)
         return VIDEO_RE.findall(reqObj.text)[0]
 
-    return playSource
+    return playSource()
```

Comment:
commented about this like in 3 diff PRs...
you sholud NOT call playsource at this stage, when you play the url, and it's a function, it gets called..
this is the way we "lazy-load" episodes, and actualy resovle them only after selection.. much less load time.